solutions/search/esql-full-text-filter-tutorial.md

```diff
@@ -0,0 +1,412 @@
+---
+navigation_title: "Search and filter with ES|QL"
+---
+
+# Tutorial: Full-text search and filtering with {{esql}}
+
+:::{tip}
+This tutorial presents examples in {{esql}} syntax. Refer to [the Query DSL version](querydsl-full-text-filter-tutorial.md) for the equivalent examples in Query DSL syntax.
+:::
+
+This is a hands-on introduction to the basics of [full-text search](full-text.md) with Elasticsearch, also known as *lexical search*, and how to filter search results based on exact criteria. In this scenario, we're implementing a search function for a cooking blog. The blog contains recipes with various attributes including textual content, categorical data, and numerical ratings.
```

Comment:
Here we focus primarily on lexical search. where do you think we should add a note on semantic search with `match`? If possible I would add a section on semantic search in this guide - can also be done as a follow up.

Reply:
https://github.com/elastic/docs-content/pull/809/commits/6249e4a46c61b39667ed8b227fec7d232e6fd382